docs/en/ingest-management/release-notes/release-notes-8.18.asciidoc

```diff
@@ -33,6 +33,45 @@ Also see:
 [[release-notes-8.18.7]]
 == {fleet} and {agent} 8.18.7
 
+[discrete]
+[[known-issues-8.18.7]]
+=== Known issues
+
+[[known-issue-2978-8-18-7]]
+.Failed upgrades leave {agent} stuck until restart
+[%collapsible]
+====
+
+This known issue applies to {agent} 8.18.7 and 9.0.7. {agent} versions 8.19.x and 9.1.x are not affected.
+
+On September 17, 2025, a known issue was discovered that can cause {agent} upgrades to get stuck if an upgrade attempt fails early. This happens because the coordinator's overrideState remains set, leaving the agent in a state that appears to be upgrading.
+
+**Conditions**
+
+This issue is triggered if the upgrade fails during one of the early checks inside Coordinator.Upgrade, for example:
+
+- The agent is not upgradeable
+- Capabilities check denies the upgrade
+- Most commonly: When {agent} is tamper-protected and Endpoint returns an error during action proxying, for example, because the upgrade action signature is invalid, missing, or fails verification. This causes the coordinator's override state to be stuck.
```

Comment:
```suggestion
- Most commonly: When {agent} is tamper-protected and Endpoint fails to validate that the upgrade action was correctly signed by Kibana to allow the upgrade, for example, because the signature is missing, invalid, or the connection between {agent} and endpoint was interrupted. This causes the agent coordinator's override state to become stuck until the agent is restarted.
```

Trying to reword this a bit so users can understand the specific circumstances more clearly.

Reply:
@cmacknz, what do you this about this tweak to your suggestion?  I'm trying to capture the extra context that you provided, while making the content easier to read and parse. 

_"When {agent} is tamper-protected, Endpoint must validate that the upgrade action was correctly signed by Kibana to allow the upgrade. If the signature is missing, invalid, or the connection between {agent} and Endpoint was interrupted, the validation fails. This causes the agent coordinator's override state to become stuck until the agent is restarted."_

Reply:
Looks great, thanks!